src/main/java/net/sf/jabref/logic/importer/fileformat/MrDLibImporter.java

```diff
@@ -0,0 +1,267 @@
+                        htmlSnippetSingle = null;
+                        htmlSnippetSingleRank = -1;
+                    }
+                    if (qName.equalsIgnoreCase("snippet")
+                            && attributes.getValue(0).equalsIgnoreCase("html_fully_formatted")) {
+                        snippet = true;
+                    }
+                    if (qName.equalsIgnoreCase("suggested_rank")) {
+                        rank = true;
+                    }
+
+                }
+
+                @Override
+                public void endElement(String uri, String localName, String qName) throws SAXException {
+                    if (qName.equalsIgnoreCase("related_article")) {
+                        rankedBibEntries.add(new RankedBibEntry(currentEntry, htmlSnippetSingleRank));
+                        currentEntry = new BibEntry();
+                    }
+                }
+
+                @Override
+                public void characters(char ch[], int start, int length) throws SAXException {
+
+                    if (authors) {
+                        currentEntry.setField("author", new String(ch, start, length));
+                        authors = false;
+                    }
+                    if (published_in) {
+                        currentEntry.setField("journal", new String(ch, start, length));
+                        published_in = false;
+                    }
+                    if (title) {
+                        currentEntry.setField("title", new String(ch, start, length));
+                        title = false;
+                    }
+                    if (year) {
+                        currentEntry.setField("year", new String(ch, start, length));
+                        year = false;
+                    }
+                    // -------------
+                    if (rank) {
+                        htmlSnippetSingleRank = Integer.parseInt(new String(ch, start, length));
+                        rank = false;
+                    }
+                    if (snippet) {
+                        currentEntry.setField("html_representation", new String(ch, start, length));
+                        snippet = false;
+                    }
+
+                }
+
+            };
+
+            try {
+                InputStream stream = new ByteArrayInputStream(recommendations.getBytes());
+                saxParser.parse(stream, handler);
+            } catch (Exception e) {
+                LOGGER.error(e.getMessage(), e);
+            }
+            rankedBibEntries.sort(new Comparator<RankedBibEntry>() {
+
+                @Override
+                public int compare(RankedBibEntry o1, RankedBibEntry o2) {
+                    return o1.rank.compareTo(o2.rank);
+                }
+            });
+
+
+            bibEntries = rankedBibEntries.stream().map(e -> e.entry).collect(Collectors.toList());
+        } catch (ParserConfigurationException e) {
```

Comment:
Use `ParserConfigurationException  | ParserConfigurationException ` to combine the catch